### PR TITLE
Change listCommit call logic

### PR DESCRIPTION
**Pull request by ad-47**

### CODE REVIEW

File: backend/src/main/java/ai/verta/modeldb/versioning/CommitDAORdbImpl.java [hunk 1]
```diff
@@ -164,11 +152,11 @@ private String generateCommitSHA(String blobSHA, Commit commit, long timeCreated
   }
 
   private boolean commitRepositoryMappingExists(
-      Session session, String commitHash, Long repositoryId) {
+      Session session, List<String> commitHash, Long repositoryId) {
```
Comment (ravishetye) [inline backend/src/main/java/ai/verta/modeldb/versioning/CommitDAORdbImpl.java]: not sure why this change was required.

Comment (ad-47) [reply backend/src/main/java/ai/verta/modeldb/versioning/CommitDAORdbImpl.java]: reverted changes
